sqlparser/lib/src/analysis/schema/column.dart

```diff
@@ -103,6 +103,73 @@ class TableColumn extends Column {
   }
 }
 
+/// A column that is part of a table.
+class ViewColumn extends Column {
```

Comment:
I think the best way to represent a `ViewColumn` would be to make this mix in `DelegatedColumn`, where the `innerColumn` is a field holding the column from the select statement used in the create view statement.

We can then remove pretty much everything from this class except from `humanReadableDescription`.

Reply:
thanks, I changed it as you described.